File: app/src/main/java/com/soya/launcher/ui/fragment/GuideGradientFragment.java
package com.soya.launcher.ui.fragment;

import android.os.Bundle;

public class GuideGradientFragment extends AbsGradientFragment{
    public static GuideGradientFragment newInstance() {
        
        Bundle args = new Bundle();
        
        GuideGradientFragment fragment = new GuideGradientFragment();
        fragment.setArguments(args);
        return fragment;
    }
}

File: app/src/main/java/com/soya/launcher/ui/fragment/GuideGradientFragment.kt
package com.soya.launcher.ui.fragment

import android.os.Bundle
import android.view.View
import com.shudong.lib_base.base.BaseViewModel
import com.soya.launcher.databinding.FragmentGradientBinding

object GuideGradientFragment : AbsGradientFragment<FragmentGradientBinding,BaseViewModel>() {
    fun newInstance(): GuideGradientFragment {
        val args = Bundle()

        val fragment = GuideGradientFragment
        fragment.arguments = args
        return fragment
    }

    override fun onClick(p0: View?) {

    }
}
